docs/docusaurus.config.ts

```diff
@@ -74,7 +74,23 @@ const config: Config = {
 	// TODO: consider re-enabling after the following issue is resolved:
 	// <https://github.com/Azure/static-web-apps/issues/1036>
 	// trailingSlash: false,
-	plugins: ["docusaurus-plugin-sass"],
+	plugins: [
+		"docusaurus-plugin-sass",
+		function addTrustedTypesOutput() {
```

Comment:
This looks great!

Could you:

1. Name it reflecting the fact that it's a docusaurus plugin (maybe just call it `trustedTypesPlugin`?)
2. Link the docusaurus plugin APIs https://docusaurus.io/docs/api/plugin-methods/lifecycle-apis as well as webpack https://webpack.js.org/configuration/output/#outputtrustedtypes